### PR TITLE
increase tolerance to back-off timing tests

### LINKED ISSUE #16248: E2E flake: Pods should have their auto-restart back-off timer reset on image update
http://kubekins.dls.corp.google.com:8081/job/kubernetes-pull-build-test-e2e-gce/14168/#

```
Stacktrace

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/pods.go:913
Oct 24 23:11:13.505: updating image did not reset the back-off value in pod=pod-back-off-image/back-off d3=29s d2=25s d1=50s
```

### PR DESCRIPTION
**Pull request by samsabed**
Increase the tolerance to account for system latency
fixes: #16248 #15800

### CODE REVIEW

**Comment by ixdy** [comment]
ok to test

**Comment by dchen1107** [comment]
LGTM

**Comment by dchen1107** [comment]
ok to test

File: test/e2e/pods.go [hunk 1]
```diff
@@ -856,7 +856,7 @@ var _ = Describe("Pods", func() {
 		delay1, delay2 := startPodAndGetBackOffs(framework, pod, podName, containerName, buildBackOffDuration)
 		delay1 += 1 // divide by zero
 		ratio := float64(delay2) / float64(delay1)
-		if math.Floor(ratio) != 2 && math.Ceil(ratio) != 2 {
+		if ratio < 1 || ratio > 4 {
```
Comment (lavalamp) [inline test/e2e/pods.go]: This won't fix all flakes; I put some examples in #15800.

**Comment by samsabed** [comment]
#16339 removes the test and tweaks the values for #16248 , superseding this

**Comment by dchen1107** [comment]
Close this PR to in favor of #16339

**Pull request closed without merging**